picobot/painter.py

```diff
@@ -0,0 +1,224 @@
+from PIL import Image, ImageDraw, ImageFont
+from .config import ROOT_DIR
+
+# for testing purpose only
+# ROOT_DIR = '/home/diogo/Documents/CCO/Python/pico-bot/picobot'
+
+IMG_DIR = ROOT_DIR + '/images/'
+IMG_NAME = 'img'
+AVATAR_MASK_NAME = 'avatar_mask.png'
+
+AVATAR_SIZE = 42
+MARGIN = 4
+PADDING = 10
+MSG_PADDING_H = 8
+MSG_PADDING_V = 6
+TIME_PADDING_H = 8
+TIME_PADDING_V = 6
+BOX_HEIGHT = 54
+BOX_MIN_WIDTH = 160
+BOX_MAX_WIDTH = 264
+BOX_RADIUS = 10
+BOX_COLOR = "#182533"
+FONT_SIZE = 14
+TITLE_COLOR = "#338cf3"
+TEXT_COLOR = "#dddddd"
+TIME_COLOR = "#6A7B8C"
+LINE_SPACE = 10
+
+
+def draw_balloon(
+    img_draw: ImageDraw.Draw, xy: list, fill=None, width=0
```

Comment:
`xy: list` should be `xy: List[int]` (`from typing import List`)

Actually a `Rect` NamedTuple would fit even better.